Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.h

```diff
@@ -144,6 +145,7 @@ class ModelProcessModelPlayerProxy final
     void computeTransform();
     void applyEnvironmentMapDataAndRelease();
     void applyStageModeOperationToDriver();
+    bool stageModeInteractionInProgress();
```

Comment:
Seems like this could be marked `const` as well, since it's just a getter.

```suggestion
    bool stageModeInteractionInProgress() const;
```